h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstNaRowCnt.java

```diff
@@ -0,0 +1,50 @@
+package water.rapids.ast.prims.reducers;
+
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.rapids.Env;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValNum;
+import water.rapids.Val;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This method counts the number of rows that contain NAs in a frame.  It does not count
+ * each occurrence of the NA but rather the number of rows that contain at least one NA.
+ */
+public class AstNaRowCnt extends AstPrimitive {
+  @Override
+  public String[] args() {
+    return new String[]{"frame"};
+  }
+
+  @Override
+  public String str() {
+    return "naRowCnt";
+  }
+
+  @Override
+  public int nargs() {
+    return 1 + 1;
+  }  // (naCnt fr)
+
+  @Override
+  public Val apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
+    Frame fr = stk.track(asts[1].exec(env)).getFrame();
```

Comment:
This is really slow and will pull data on a single node. Might be ok for datasets with small number of rows but once you add it someone will use it on a big one some item in the future and get OOM.  
I'd prefer if you can do an MRTask here (the logic is the same, you just do it per-chunk and sum the results up in reduce)..